cucumber/features/steps/pan.rb

```diff
@@ -291,6 +291,9 @@ def flick_broken?
 end
 
 And(/^I have scrolled to the top of the Companies table$/) do
+  # Skip this step since StatusBar was deprecated on iOS 13+
+  next if ios_gte?("13.0")
+
```

Comment:
We still have to scroll to the top of the view, right?

I think we need to scroll in a loop until we see the first row.

For now, let's branch:

```
if ios_gte?("13.0")
  < scroll to top >
else
  < touch the top of the screen >
end
```

I think I asked this already, but what happens if you touch the top of the screen at (4, 4) on iOS 13?  Does the table scroll to the top?

Reply:
Actually, we don't have to scroll on top since this test doesn't scroll to bottom. Scrolling doesn't make sense here.

As for your question about touching screen at (4, 4)-  nothing will happen on iOS 13

Reply:
> scroll to top

This step doesn't but the previous step might.  For example:

```
Scenario: Swipe to delete table view row
And I am looking at the Everything's On the Table page
Then I can flick to the bottom of the Companies table <== at the bottom
And I can swipe to delete the Windows row  

Scenario: Reorder table view row
And I am looking at the Everything's On the Table page
And I have scrolled to the top of the Companies table  <== scroll to top
When I touch the Edit button, the table is in Edit mode
Then I move the Android row above the Apple row  <== if we don't scroll to the top, we don't see these rows.
```

Reply:
Steps from other scenarios don't affect this scenario because `Companies` table is closed and opened again between tests so it is on top at the begin of this scenario always.
I tried to run the whole pan.feature suite and it passed without errors.

Please let me know if we need to add scroll to top anayway

Reply:
In that case, let's remove the "I have scrolled to the top of the Companies table" step.

Reply:
Done